pkg/bpf/map_linux.go

```diff
@@ -781,15 +787,15 @@ func (m *Map) DumpIfExists(hash map[string][]string) error {
 }
 
 func (m *Map) Lookup(key MapKey) (MapValue, error) {
-	m.lock.RLock()
-	defer m.lock.RUnlock()
-
-	value := key.NewValue()
-
 	if err := m.Open(); err != nil {
```

Comment:
Same here. Could we use `m.open` and move it aftger the lock/defer unlock below (line 796/797)?